go.mod

```diff
@@ -5,12 +5,17 @@ go 1.20
 require (
 	github.com/AlekSi/pointer v1.0.0
 	github.com/aws/aws-sdk-go v1.37.8
```

Comment:
can we free ourselves of all aws-sdk-go v1?

Reply:
I tried getting rid of `awserr` lib but there is really no equivalence in sdk v2, the most closed one would be `smithy-go`, but that would still require us making some minor logic changes in the testing file. I can definitely give it another try and you can take a look

Reply:
Okay so I freed reflector pkg from v1, but I then realized that there are much more places using v1, e.g., leger_monitor which uses ecs client. I don't think it's gonna be a trivia task to completely free this repo from v1, would you think that's acceptable?